ultraLatency.py

```diff
@@ -117,12 +114,10 @@
 
 N_EVENTS = Nev[0]
 
-mask = options.vfatmask
-
 try:
-    writeAllVFATs(ohboard, options.gtx, "ContReg0",    0x37, mask)
+    writeAllVFATs(ohboard, options.gtx, "ContReg0",    0x37, options.vfatmask)
     writeAllVFATs(ohboard, options.gtx, "ContReg2",    ((options.MSPL-1)<<4))
-    writeAllVFATs(ohboard, options.gtx, "VThreshold2", options.vt2, mask)
+    writeAllVFATs(ohboard, options.gtx, "VThreshold2", options.vt2, options.vfatmask)
```

Comment:
why this change? is it not clear what `mask` was?

Reply:
See old line 120, a new variable `mask` was being used to store `options.vfatmask` which I felt was unnecessary.

Reply:
No, I saw what you did, I just wasn't clear on why you felt it was necessary (unnecessary).
* If you made separate commits (which would have been best) for the two changes you made (debug handling and `mask`), can you `cherry-pick` out the one related to the `mask`
* If not from a separate commit, then please revert the `mask` change
Then, you can create an issue if you feel it's worth changing the `mask` part